modules/apps/travel-distance-wfrp4e.js

```diff
@@ -0,0 +1,134 @@
+  }
+
+    /**
+    * This method either display the distances between 2 towns (ie when toTown is set) or display a list of destinations
+    * @param {string} fromTown
+    * @param {string} toTown
+    * @returns 
+    */
+   static displayTravelDistance( fromTown, toTown ) {
+    
+    let message = "";
+
+    //console.log("TRAVEL ...", fromTown, toTown );
+    if ( toTown ) {
+      fromTown = fromTown.toLowerCase();
+      toTown = toTown.toLowerCase();
+      for ( var travel of this.travel_data) {
+        //console.log("    ", travel);
+        if ( travel.from.toLowerCase() == fromTown && travel.to.toLowerCase() == toTown ) {
+          message += `<p>${game.i18n.format("TRAVEL.TravelMessageBase", travel)}`;
+          if ( travel.road_distance != "" ) {
+            travel.road_horse_days = this.roundDuration( travel.road_days * 0.9);
+            travel.road_feet_days  = this.roundDuration( travel.road_days * 1.2 );
+            travel.road_danger_string = this.dangerToString( travel.road_danger );
+            travel.road_danger_feet_string = this.dangerToString( travel.road_danger + "!" ); // Increase danger level by feet
+            message += `${game.i18n.format("TRAVEL.TravelMessageRoad", travel)}`;                    
+          }
+          if ( travel.river_distance != "" ) {
+            travel.river_danger_string = this.dangerToString( travel.river_danger );
+            message += `${game.i18n.format("TRAVEL.TravelMessageRiver", travel)}`;                    
+          }
+          if ( travel.sea_distance != "" ) {
+            travel.sea_danger_string = this.dangerToString( travel.sea_danger );
+            message += `${game.i18n.format("TRAVEL.TravelMessageSea", travel)}`;                    
+          }
+          message += "</p>";
+          //console.log("  Destination : ", travel.to );
+        }
+      }
+    
+    } else if ( fromTown && fromTown == "help") {
+      message += `<p>${game.i18n.localize("TRAVEL.Helper")}</p>`;
+
+    } else if ( fromTown) {
+      fromTown = fromTown.toLowerCase();
+      message += `<h3>${game.i18n.localize("TRAVEL.TownPrompt")}</h3>`
+      for ( var travel of this.travel_data) {
+        //console.log("    ", travel);
+        if ( travel.from.toLowerCase() == fromTown ) {
+          message += `<div><a class = "travel-click" data-fromtown="${travel.from}" data-totown = "${travel.to}"><i class="fas fa-list"></i> ${travel.to}</a></div>`
+        }
+      }
+    } else {
+      message += `<h3>${game.i18n.localize("TRAVEL.TownOriginPrompt")}</h3>`
+      let uniqTown = {}
+      for ( var travel of this.travel_data) {
+        if ( uniqTown[travel.from] == undefined ) {
+          uniqTown[travel.from] = 1; // Already added in the list
+          message += `<div><a class = "travel-click" data-fromtown="${travel.from}"><i class="fas fa-list"></i> ${travel.from}</a></div>`
+        }
+      }
+    }
+    ChatMessage.create( WFRP_Utility.chatDataSetup(message, "roll") );      
```

Comment:
Can you remove the `modeOverride`  so it takes the default roll mode for visibility?

Reply:
Fixed, with distance tables updated and duration with heavy and light horses